backend/src/Controller.ts

```diff
@@ -124,6 +125,8 @@ class Controller {
   preferences = preferences.set
 
   syncBackend = async () => {
+    environment.checkOob()
+    await lan.getInterfaces()
```

Comment:
The sync command is meant to send initial state up to the frontend, it should not trigger these checks, the checks should start with the application startup and could either be moved to Application.ts or the class's constructor. Also they are duplicated above...

Reply:
I updated to work like lan interfaces.  Let me know if there is a better approach.

Reply:
Looks good!